plugins/confluence/commands/sync-meeting-notes.md

```
@@ -0,0 +1,143 @@
+   mcp__mcp-atlassian__confluence_add_label(
+       page_id="<page-id>",
+       name="meeting-2026-05"
+   )
+   ```
+2. Add a `meeting-notes` label for discoverability.
+3. Report the page URL and summary of extracted items (action count, decision count).
+
+## Usage Examples
+
+1. **Publish grooming notes after running `/jira:grooming`:**
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)